code/modules/paperwork/prefab_papers/wey_yu/liaison/ops_report.dm

```diff
@@ -0,0 +1,138 @@
+	dat += ".header-subtitle {"
+	dat += "font-size: 15px;"
+	dat += "text-transform: uppercase;"
+	dat += "letter-spacing: 1.5px;"
+	dat += "margin-top: 4px;"
+	dat += "}"
+	dat += ""
+	dat += ".message-body-text {"
+	dat += "text-align: left;"
+	dat += "font-size: 17px;"
+	dat += "margin-bottom: 35px;"
+	dat += "}"
+	dat += ""
+	dat += ".message-signature-text {"
+	dat += "text-align:right;"
+	dat += "font-size:15px;"
+	dat += "margin-bottom: 20px;"
+	dat += "}"
+	dat += ".signature-name {"
+	dat += "font-size: 20px;"
+	dat += "font-weight: 800;"
+	dat += "margin-bottom: 0px;"
+	dat += "}"
+	dat += ""
+	dat += ".signature-title{"
+	dat += "font-size: 14;"
+	dat += "text-transform: uppercase;"
+	dat += "margin-top: 2px;"
+	dat += "}"
+	dat += ""
+	dat += ".message-signature-small{"
+	dat += "font-size: 16px;"
+	dat += "font-style: italic;"
+	dat += "line-height: 1.4;"
+	dat += "margin-top: -6px;"
+	dat += "}"
+	dat += ""
+	dat += ".message-signature-s-small{"
+	dat += "font-size: 10px;"
+	dat += "font-style: italic;"
+	dat += "line-height: 1.4;"
+	dat += "margin-top: -20px;"
+	dat += "}"
+	dat += ""
+	dat += ".disclaimer{"
+	dat += "font-size: 8px;"
+	dat += "line-height: 1.2;"
+	dat += "}"
+	dat += ""
+	dat += "</style>"
+	dat += ""
+	dat += "<div id='width-container'>"
+	dat += "<div id='wy-fax-logo'>"
+	dat += "<img src='[asset.get_url_mappings()["logo_wy.png"]]' alt='WY Logo'/>"
+	dat += "</div>"
+	dat += ""
+	dat += "<!--Header Info here-->"
+	dat += "<div class='message-header-text'>"
+	dat += "<p class='header-title'>Liaison Operations Report</p>"
+	dat += "<p class='header-subtitle'>WY435</p>"
+	dat += "<hr style='margin-top:5px; margin-bottom:15px;''>"
+	dat += "</div> <!-- /Header Info -->"
+	dat += ""
+	dat += "<!-- Message -->"
+	dat += "<div class='message-body-text'>"
+	dat += "<p>Index: <span class=\"paper_field\"></span><br>"
+	dat += "Date: <font face=\"Times New Roman\"><i>[time2text(REALTIMEOFDAY, "Day DD Month [GLOB.game_year]")]</i></font><br>"
+	dat += "To: <span class=\"paper_field\"></span><br>"
+	dat += "Subject: <span class=\"paper_field\"></span></p>"
+	dat += "<p><ul><span class=\"paper_field\"></span></ul></p>"
+	dat += "<HR>"
+	dat += "<p>Liaison Name: <span class=\"paper_field\"></span><br>"
+	dat += "Signature: <span class=\"paper_field\"></span></p>"
+	dat += "</div> <!-- /message-body-text -->"
+	dat += ""
+	dat += "<div class='message-signature-text'>"
+	dat += "<p class='disclaimer'>This message and any files transmitted with it are confidential and intended solely for the use of the individual or entity to whom they are addressed. This message contains confidential information and is intended only for addressee, if you are not the aforementioned addressee you should not disseminate, distribute or copy this missive. If you are not the intended recipient, you are notified that disclosing, copying, distributing or taking any action in reliance on the contents of this information is strictly prohibited.</p>"
```

Comment:
```only for the addressee. If you are not the aforementioned addressee, you should not disseminate```...
add article "the" before addressee. end sentence, start new instruction; comma after "aforementioned addressee", clause stip.